modules/RTC/TraceablePeerConnection.js

```diff
@@ -1516,7 +1484,12 @@ TraceablePeerConnection.prototype.addTrack = function(track, isInitiator = false
     const webrtcStream = track.getOriginalStream();
 
     try {
-        this.tpcUtils.addTrack(track, isInitiator);
+        transceiver = this.tpcUtils.addTrack(track, isInitiator);
```

Comment:
This try block feels too wide no? What is the operation that can throw an error? Ideally we keep it as narrow as possible.